src/backends/caffe/caffelib.cc

```diff
@@ -2304,6 +2419,65 @@ namespace dd
 		  vrad.push_back(outseq);
 		}
 	    }
+           else if (typeid(inputc) == typeid(CSVTSCaffeInputFileConn)) // timeseries
+             {
+               int slot = findOutputSlotNumberByBlobName(_net,"OUTPUT");
```

Comment:
Same as above, maybe find a better name than `OUTPUT`

Reply:
changed to rnn_pred